download/index.md

```diff
@@ -89,11 +89,11 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
 * Redis Stack on [Dockerhub](https://hub.docker.com/u/redis): [x86_64 and arm64](https://hub.docker.com/r/redis/redis-stack)
 * Redis Stack server on [Dockerhub](https://hub.docker.com/u/redis): [x86_64 and arm64](https://hub.docker.com/r/redis/redis-stack-server)
 
-See the [release notes for 7.2.0-v6](https://github.com/redis-stack/redis-stack/releases/tag/v7.2.0-v6).
+See the [release notes for 7.2.0-v6](https://github.com/redis-stack/redis-stack/releases/tag/7.2.0-v6).
 
 ### Older Redis Stack Versions
 
-Redis Stack 6.2.6 introduces many new features for Search and Query, including support for [wildcard queries](https://redis.io/docs/stack/search/reference/query_syntax/#wildcard-matching), multi-value indexing and querying, and more. This version also contains a new faster JSONPath, a new t-digest probabilistic data structure, new path-finding algorithms for Graph, support for gap filling for Time series, and other features and improvements.
+Redis Stack 6.2.6 introduces many new features for Search and Query, including support for [wildcard queries](/docs/interact/search-and-query/advanced-concepts/query_syntax/#wildcard-queries), multi-value indexing and querying, and more. This version also contains a new faster JSONPath, a new t-digest probabilistic data structure, new path-finding algorithms for Graph, support for gap filling for Time series, and other features and improvements.
```

Comment:
@dwdougherty , let's use this link?
https://redis.io/docs/interact/search-and-query/advanced-concepts/query_syntax/#wildcard-queries